src/main/java/org/folio/consortia/service/FolioTenantService.java

```diff
@@ -81,6 +93,47 @@ protected boolean tenantExists() {
     );
   }
 
+  private void updateLocalTenantShadowSystemUsers() {
+    if (!consortiaConfigurationService.isCentralTenantConfigurationExists()) {
+      log.info("The first time module is being installed");
+      return;
+    }
+
+    log.debug("updateLocalTenantShadowSystemUsers:: Trying  to update shadow user permissions");
+    String requestingTenant = folioExecutionContext.getTenantId();
+    String centralTenantId = consortiaConfigurationService.getCentralTenantId(requestingTenant);
```

Comment:
In case if central tenant not added, we will have null and so can have only 1 DB call instead of 2

Reply:
I see, exception is throwing, okay lets leave as is

Reply:
That was my initial plan, but I found that error is being thrown when there is no configuration. I also though about one db call fetch central tenant but I found that there will be duplicate method.
![image](https://github.com/folio-org/mod-consortia/assets/113523904/59ab3cd7-d967-4f18-8c41-de5d3fa53e26)